src/Test/Utilities/Desktop/TraceListener.cs

```diff
@@ -1,81 +0,0 @@
-﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
-
-using System;
-using System.Diagnostics;
-
-namespace Microsoft.CodeAnalysis
-{
-#if DEBUG
-    // To enable, add to <listeners> in *.exe.config, specifying the assembly-qualified
```

Comment:
It would be good to keep this comment explaining how one can take advantage of this type.